qsv/INCOMING/unemployment.luau

```diff
@@ -0,0 +1,14 @@
+BEGIN {
+    csv_indexed = qsv_autoindex();
+    unemployment_lookup_headers = qsv_register_lookup("unemployment2023", "https://data.dathere.com/dataset/e0bcff0e-e21f-42bf-829d-8cbe059cf954/resource/f56a104b-e943-4f49-8755-0891d3284669/download/2023data.csv", 10000)
+}
+
+local month_name = col["Month"]

```

Comment:
Seems like we're missing the name of place where we're fetching the unemployment rate for. Or maybe i'm wrong. not sure how this works. 

Could you attach a test dataset for this?

Thanks 👍